UnityAds/Video/UADSAVPlayer.m

```diff
@@ -123,6 +123,7 @@ - (void)stopPrepareTimeoutTimer {
 
 - (void)onPrepareTimeoutListener:(NSNotification *)notification {
     UADSLogError(@"Video prepare timeout");
+    self.prepareTimeoutTimer = NULL;
```

Comment:
Instead of setting the `self.prepareTimeoutTimer` variable you could call `stopPrepareTimeoutTimer()` function, which will invalidate and set to `NULL` if it's needed.